src/style.css

```diff
@@ -1 +1,85 @@
 /* Styles go here */
+/* !!!!basic styles!!! */
+body,
+header,
+nav,
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a,
+a:hover,
+a:visited,
+a:active {
+  outline: none;
+  text-decoration: none;
+  color: inherit;
+}
+
+img {
+  vertical-align: middle;
+}
+
+ul {
+  list-style: none;
+}
+
+body {
+  height: 100%;
+  font-family: "Open Sans", sans-serif;
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/59209337/84c6679f-12d3-4d7c-b800-18b874634437)

please, make sure that all points from the checklist are completed